web-frontend/src/pages/TransferCoins/SendCoins/Send.tsx

```diff
@@ -1,11 +1,10 @@
-import Intl from '../../../i18n/i18n';
 import { parseForm } from '../../../utils/parseForm';
 import { useState, ChangeEvent, FormEvent } from 'react';
 import {
   formatStandard,
-  reverseFormatStandard,
-  checkRecipientFormat,
   Unit,
+  validate,
+  validateAmount,
```

Comment:
What's the difference between them!?

Reply:
validate checks both the amount and the recipient

Reply:
Ok! Great! In this case why we have validateAmount then!? 🤔

Reply:
For the selection of the fees, there is no field for recipient there

Reply:
Humm... I see! Ok! 

In future we must to review anyway how we improve that.